project-base/assets/styles/frontend/common/components/box/filter.less

```diff
@@ -59,116 +59,88 @@
         }
     }
 
-    &__slider {
-        &__wrap {
-            font-size: 0;
-            margin-top: 15px;
-
-            &__range {
-                .is-no-js & {
-                    display: none;
-                }
-            }
-        }
-    }
-
     &__price {
-        width: 100%;
-        padding: @box-filter-padding;
-
-        font-size: 0;
+        display: flex;
+        flex-wrap: wrap;
+        padding-bottom: 14px;
+        padding-top: 4px;
+
+        &__slider {
+            width: 100%;
+            margin-bottom: 16px;
+        }
 
         &__item {
-            width: @box-filter-input-width;
-            display: inline-block;
-            line-height: @form-item-height;
-            margin-top: 15px;
+            display: flex;
+            width: @box-filter-input-width + 30px;
+            align-items: center;
+
+            @media @query-lg {
+                width: 50%;
+            }
 
             .input {
                 height: 40px;
                 width: @box-filter-input-width;
                 text-align: center;
+                padding: 0 3px;
 
-                font-size: 15px;
                 font-weight: 600;
                 color: @color-0;
 
                 &::-webkit-input-placeholder { /* Firefox 18- */
-                    opacity: 1;
-
                     color: @color-0;
                     font-style: normal;
                     font-weight: 500;
                 }
 
                 &::-moz-placeholder { /* Firefox 18- */
-                    opacity: 1;
-
                     color: @color-0;
                     font-style: normal;
                     font-weight: 500;
                 }
 
                 &:-ms-input-placeholder { /* Firefox 18- */
-                    opacity: 1;
-
                     color: @color-0;
                     font-style: normal;
                     font-weight: 500;
                 }
 
                 &:-moz-placeholder { /* Firefox 18- */
-                    opacity: 1;
-
                     color: @color-0;
                     font-style: normal;
                     font-weight: 500;
                 }
             }
 
-            &--text {
-                width: calc(~"100% - 2*@{box-filter-input-width}");
-                padding: 0 5px;
+            &__text {
+                flex: 1;
```

Comment:
How it looks in old IE? This could be damaged. Please check it.

Reply:
It looks good on IE11.